conda.recipe/meta.yaml

```diff
@@ -44,19 +44,45 @@ requirements:
     - cytoolz >=0.8.1
 
 test:
+  source_files:
+    - conftest.py
+    - tests
+    - setup.cfg
   requires:
     - mock
     - pytest
+    - pexpect
+    - responses
   imports:
     - conda
-  # commands:
-    # TODO: need to move to a run_tests file
-    # TODO: disable these tests for now, because symlinking conda still giving issues
-    # - . $PREFIX/etc/profile.d/conda.sh && conda --version               [unix]
-    # - . $PREFIX/bin/conda info                                          [unix]
-    # - "%PREFIX%/Scripts/conda --version"                        # [win]
-    # - "%PREFIX%/Scripts/conda info"                             # [win]
-    # - py.test -m "not integration and not installed" $SRC_DIR/tests $SRC_DIR/conda
+    - conda_env
+  commands:
+{% if on_win %}
+    - conda activate base
+    - FOR /F "delims=" %%i IN ('python -c "import sys; print(sys.version_info[0])"') DO set "PYTHON_MAJOR_VERSION=%%i"
+    - set TEST_PLATFORM=win
+    - FOR /F "delims=" %%i IN ('python -c "import random as r; print(r.randint(0,4294967296))"') DO set "PYTHONHASHSEED=%%i"
+    - set
+    - conda info
+    - py.test tests -m "not integration and not installed" -vv
+    - conda create -y -p .\built-conda-test-env
+    - conda activate .\built-conda-test-env
+    - echo %CONDA_PREFIX%
+    - IF NOT "%CONDA_PREFIX%"=="%CD%\built-conda-test-env" EXIT /B 1
```

Comment:
For whatever reason, conda-build doesn't seem to want to run the above on Windows.  It doesn't have a problem running the below on macOS though.